superdesk/metadata/item.py

```diff
@@ -155,6 +155,17 @@
             }
         }
     },
+    'company_codes': {
```

Comment:
Can you remove `company_codes` definition from https://github.com/superdesk/superdesk-core/blob/master/apps/archive/common.py#L138?